src/commands/deploy/deploySmartWearable.ts

```diff
@@ -0,0 +1,234 @@
+      errors
+    )
+    throw new Error(`Invalid asset.json (${assetJsonPath})`)
+  }
+
+  let collectionId = (assetJson as any)?.collectionId || ''
+  if (!uuid.validate(collectionId)) {
+    const results = await inquirer.prompt({
+      type: 'confirm',
+      name: 'continue',
+      message: warning(
+        `You haven't assigned a valid collection ID. Using collection can help to group wearables and test them. Do you want to assign one? It should already exist.`
+      )
+    })
+
+    if (results.continue) {
+      while (true) {
+        const answers = await inquirer.prompt([
+          {
+            type: 'input',
+            name: 'uuid',
+            message: 'Destination collection (a valid UUID, empty to skip):'
+          }
+        ])
+
+        const ans = (answers.uuid as string) || ''
+        if (ans.length === 0) {
+          console.info(`Skipping collection ID.`)
+          break
+        } else if (uuid.validate(answers.uuid)) {
+          collectionId = answers.uuid
+          console.info(`Using collection ID: ${collectionId}`)
+          const newAssetJson = { ...assetJson, collectionId }
+          await writeFile(assetJsonPath, JSON.stringify(newAssetJson, null, 2))
+          break
+        } else {
+          console.error('This is not a valid UUID.')
+        }
+      }
+    } else {
+      console.info(`Skipping collection ID.`)
+    }
+  } else {
+    console.info(`Using collection ID : ${collectionId}`)
+  }
+
+  spinner.create('Building smart wearable')
+
+  const assetBasicConfig = assetJson as sdk.AssetJson
+  const thumbnailContent = await readFile(
+    path.resolve(project.getProjectWorkingDir(), assetBasicConfig.thumbnail)
+  )
+
+  let originalFilesToIgnore = await project.getDCLIgnore()
+  if (originalFilesToIgnore === null) {
+    originalFilesToIgnore = await project.writeDclIgnore()
+  }
+
+  const content: Record<string, Uint8Array> = {}
+  const files: IFile[] = await project.getFiles(originalFilesToIgnore)
+  for (const file of files) {
+    content[file.path] = new Uint8Array(file.content.buffer, 0)
+  }
+  content['thumbnail.png'] = content[assetBasicConfig.thumbnail]
+  delete content['asset.json']
+
+  const item = new ItemFactory<Uint8Array>()
+    .newItem({
+      id: assetBasicConfig.id,
+      name: assetBasicConfig.name,
+      description: assetBasicConfig.description,
+      rarity: assetBasicConfig.rarity
+    })
+    .withCategory(assetBasicConfig.category as any)
+    .withThumbnail(thumbnailContent)
+    .withRepresentation(
+      assetBasicConfig.bodyShape as any as BodyShapeType,
+      assetBasicConfig.model,
+      content,
+      {
+        triangles: 0,
+        materials: 0,
+        meshes: 0,
+        bodies: 0,
+        entities: 0,
+        textures: 0
+      }
+    )
+
+  if (collectionId) {
+    item.withCollectionId(collectionId)
+  }
+
+  const builtItem = await item.build()
+
+  spinner.succeed('Smart wearable built!')
+
+  dcl.on('link:ready', (url) => {
```

Comment:
could we move this to a fn ? It's the same logic that we have at deployScene.
I think we should go for a fn approach and wrap all the common fns 

```suggestion
  dcl.on('link:ready', (url) => {
     openBrowser()
  })
```